app/controllers/medications_controller.rb

```diff
@@ -51,29 +51,36 @@ def edit
   # POST /medications.json
   def create
     @medication = Medication.new(medication_params)
-    respond_to do |format|
-      if @medication.save
-        save_refill_to_google_calendar(@medication)
+    return unless save_refill_to_google_calendar(@medication)
+
+    if @medication.save
+      respond_to do |format|
```

Comment:
So the respond_to blocks starting at lines 57 and 62 are similar. We could move this out into a helper method. This would stop Codeclimate from complaining lol.